src/insns/amo_32bit.adoc

```diff
@@ -48,11 +36,11 @@ Standard atomic instructions, authorised by the capability in <<ddc>>.
 
 include::atomic_exceptions.adoc[]
 
-Prerequisites for CAMO<OP>.W, CAMO<OP>.D::
-{cheri_base_ext_name}
+Prerequisites for AMO<OP>.W, AMO<OP>.D::
+{cheri_base_ext_name}, and A, or +
+{cheri_legacy_ext_name}, and A
 
 Prerequisites for AMO<OP>.W, AMO<OP>.D::
```

Comment:
This looks wrong to me now since there is an empty duplicate heading.

Maybe the header should
```suggestion
Prerequisites for Capability Mode AMO<OP>.W, AMO<OP>.D::
{cheri_base_ext_name}, and A

Prerequisites for Legacy Mode AMO<OP>.W, AMO<OP>.D::
{cheri_legacy_ext_name}, and A
```